03_Modules/Tokens/src/module/api.ts

```diff
@@ -230,19 +236,35 @@ export class TokensModuleApi
   })
   async fetchTokens(
     @VersionNumberParam() version: VersionNumber,
-    @Param('countryCode') countryCode: string,
-    @Param('partyId') partyId: string,
-    @Paginated() paginationParams?: PaginatedParams,
+    @Body() asyncJobRequest: AsyncJobRequest,
   ): Promise<AsyncJobStatusDTO> {
-    const jobStatus = await this.tokensService.startFetchTokensByParty(
-      countryCode,
-      partyId,
-      paginationParams,
-    );
+    const jobStatus =
+      await this.tokensFetchService.startFetchTokensByParty(asyncJobRequest);
     return jobStatus.toDTO();
   }
-
-  @Get('/:countryCode/:partyId/fetch/:jobId')
+  @Post('/fetch/:jobId/:action')
+  @ResponseSchema(AsyncJobStatusDTO, {
+    statusCode: HttpStatus.OK,
+    description: 'Successful response',
+    examples: {
+      success: generateMockOcpiResponse(AsyncJobStatusDTO),
+    },
+  })
+  async fetchTokensContinue(
```

Comment:
very minor, update name to something like `fetchTokensActions` ? just wondering if "continue" is too limiting since you can stop jobs from here 🤔